main.go

```diff
@@ -43,9 +51,9 @@ func createIntCounter(meter metric.Meter, name string, description string) metri
 		)
 }
 
-func createTracesAndSpans(ctx context.Context, tracesProvides *sdktrace.TracerProvider, suites []junit.Suite) error {
-	tracer := tracesProvides.Tracer("junit2otlp")
-	meter := global.Meter("junit2otlp")
+func createTracesAndSpans(ctx context.Context, srvName string, tracesProvides *sdktrace.TracerProvider, suites []junit.Suite) error {
+	tracer := tracesProvides.Tracer(srvName)
```

Comment:
I wonder if the traceName should have a different attribute:
- Service Name where all the traces will be sent to
- Trace Name, could be the same but customisable to a different one.
What do you think?

Reply:
I like the idea of overriding the trace name, will add it here in the PR, thanks

Reply:
Added in here: https://github.com/mdelapenya/junit2otlp/pull/9/commits/0428eb487ee955a54aadf3da4085c718dc274f94

I was not able to find an env var to set the trace name, i.e. `OTEL_TRACE_NAME`, that's why I'm directly using what's coming from the flag (`junit2otlp` as fallback) Please let me know if you know how to configure that trace name using the env